data/crac-file/crac-impl/src/test/java/com/farao_community/farao/data/crac_impl/CracFileTest.java

```diff
@@ -169,17 +167,14 @@ private static SimpleCrac create() {
 
         ComplexRangeAction rangeAction2 = new ComplexRangeAction("idRangeAction2", "myRangeAction2", "operator1", usageRules, ranges, new ArrayList<>(Arrays.asList(pstRange1)));
 
-        List<Cnec> cnecs = new ArrayList<>();
-        cnecs.add(cnec1);
+        SimpleCrac crac = new SimpleCrac("idCrac", "name");
```

Comment:
Maybe Crac crac = ... (and not SimpleCrac crac = ...), and same in the other tests below?

Reply:
Yes but for some of the test I keep SimpleCrac because I use some methods implemented in SimpleCrac but which are part of Crac interface